cfgfile/schema.go

```diff
@@ -23,6 +23,7 @@ type SidecarConfig struct {
 	NodeId                     string    `config:"node_id"`
 	CachePath                  string    `config:"cache_path"`
 	LogPath                    string    `config:"log_path"`
+	CollectorConfigDirectory   string    `config:"collector_config_directory"`
```

Comment:
In analogy to `collector_binaries_whitelist` I wouldn't shorten `configuration`.